defaults/main.yml

```diff
@@ -40,3 +40,11 @@ zfs_base_defaults:
   compression: on
   relatime: on
   xattr: sa
+
+# Attributes used by zfs-auto-snapshot
+  com_sun_auto_snapshot: False
```

Comment:
Why not namespace them?

Reply:
this is a variable inside a namespaced dict
besides, the variable should be named as close as possible to the original usage in ZFS